Makefile

```diff
@@ -34,5 +34,7 @@ uninstall:
 	    rm $(LOCALPATH)/$(EXE); \
 	fi
 
-tests:
+reinstall: uninstall install
+
+tests: reinstall
```

Comment:
This seems odd; why are we running the `reinstall` target when running `make tests`?